certs_to_crlite.py

```diff
@@ -1,11 +1,209 @@
-import datetime
+# Python Standard Library
+import argparse
+import base64
+import binascii
+from collections import Counter
+from datetime import datetime
+import os
 from struct import pack
 import sys
+import time
 
+# 3rd-party libraries
 import censys.certificates
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
 from decouple import config
+
+# Local modules
 from filter_cascade import FilterCascade
 
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--path", help="Path to folder on disk with CT certs")
```

Comment:
E.g., on my machine, I ran [`ct-fetch`](https://github.com/jcjones/ct-mapreduce/tree/master/cmd/ct-fetch) with this `.conf`:
```
issuerCNList = DigiCert
logList = https://ct.cloudflare.com/logs/nimbus2021/,https://ct.cloudflare.com/logs/nimbus2020/
certPath = /Users/lcrouch/code/mozilla/ct-mapreduce/ct
```

And ran with this:
```
python certs_to_crlite.py --path /Users/lcrouch/code/mozilla/ct-mapreduce/ct
```